### PR TITLE
[US-628] UniOffice Api Usage Example

### PR DESCRIPTION
**Pull request by kellemNegasi**
This PR provides an example code on how to set verbose mode of license key usage logging. A code of simple docx editing is used here to showcase this.

### CODE REVIEW

File: license/usage-logs/main.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := unipdflicense.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		fmt.Printf("ERROR: Failed to set metered key: %v\n", err)
+		fmt.Printf("Make sure to get a valid key from https://cloud.unidoc.io\n")
+		fmt.Printf("If you don't have one - Grab one in the Free Tier at https://cloud.unidoc.io\n")
+		panic(err)
+	}
```
Comment (sampila) [inline license/usage-logs/main.go]: Probably can make a simple operation for this example?
like open and then modify docx, so we can remove this code and we also already had convert to pdf example.

**Review by sampila — CHANGES_REQUESTED** [review]
added a comment

File: license/usage-logs/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+### Metered License Key Usage Logs
+When Using unioffice it is possible to see the usage logs of license key in every run. To enable this behavior all we have to do is setting the `UsageLog` to verbose and the log level to `Info` or above as follows.
```
Comment (sampila) [inline license/usage-logs/README.md]: Probably also mention that this only available for api key based license(metered api key)

**Review by sampila — COMMENTED** [review]
added a comment

File: license/usage-logs/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+### Metered License Key Usage Logs
+When using unioffice on metered api key it is possible to see the usage logs of license key in every run. To enable this behavior all we have to do is setting the `UsageLog` to verbose and the log level to `Info` or above as follows. This is available for metered api key only.
```
Comment (sampila) [inline license/usage-logs/README.md]: ... To enable this set the `license.SetMeteredKeyUsageLogVerboseMode` to `true` and to print out into console set the `logger.SetLogger` to `Info` or higher.

File: license/usage-logs/main.go [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+
+	// Set the log level to info or higher
+	logger.SetLogger(logger.NewConsoleLogger(logger.LogLevelInfo))
+
+	// Enable the verbose mode logging
```
Comment (sampila) [inline license/usage-logs/main.go]: // Enable the usage log verbose mode to log the usages info

**Comment by kellemNegasi** [comment]
@sampila I have made some changes, can you check again.

**Review by sampila — APPROVED** [review]
Looks good to me

**Pull request merged**